alert-policies/dojo/ErrorResponse.yml

```diff
@@ -0,0 +1,36 @@
+name: Error Response
```

Comment:
Please change it to Ajax Error Response

Also, please change the file name to "AjaxErrorResponse"